src/services/createCommunity.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+const getTokenFromLocalStorage = () => {
```

Comment:
This seems to be a redundant file. Please verify which file are you using in the following component.

Reply:
Remove the extra files.